packages/astro/package.json

```diff
@@ -179,7 +179,7 @@
     "zod": "^3.22.4"
   },
   "optionalDependencies": {
-    "sharp": "^0.33.1"
+    "sharp": "0.32.6"
```

Comment:
Is the caret (`^`) removed here intentional? (Same as for the starlog example update). `^0.32.6` should still restrict within 0.32.x if a new patch is ever released in the future.

Reply:
I think it's intentional because if users have a version of Node.js lower than the one requested by sharp 0.33.0, it won't work. That's why Erika said that users are forced to install and override their locale version.

Reply:
Even with the caret, it won't use 0.33.0 by default. Users have to override to use 0.33.0 too, so I'm not sure if we need to omit it.

Reply:
Ah no, it wasn't necessarily intentional, but Sharp never goes back and fixes previous versions, especially now that 0.33.0 completely changed how the installation works. 

Happy to change it just to be safe, though

Reply:
Thanks for changing it 👍 I think in case there's a security fix it would be nice to unblock it.